ext/standard/basic_functions.stub.php

```diff
@@ -2065,6 +2107,7 @@ function crypt(#[\SensitiveParameter] string $string, string $salt): string {}
 /**
  * @return array<string, int|string>|false
  * @deprecated
+ * @compile-time-eval
  * @refcount 1
  */
 function strptime(string $timestamp, string $format): array|false {}
```

Comment:
Doesn't strptime depend on the current locale? https://man7.org/linux/man-pages/man3/strptime.3.html does mention the locale for some format string option at least.

Reply:
Good catch, even php man https://www.php.net/manual/en/function.strptime.php mentions that. I will remove the function, it is deprecated anyway. Can you please double check also the remaining functions?

Reply:
I only took a brief look now because I'm almost out of time for today (and I still have to PR something). So I'll check again tomorrow.